### PR TITLE
Fix: Nested Tabs - Normal Tabs normal styling overrides active tab's styling [ed-9013]

### PR DESCRIPTION
**Pull request by hein-obox**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: modules/nested-tabs/assets/scss/frontend.scss [hunk 1]
```diff
@@ -13,8 +13,6 @@
 	--n-tabs-display: flex;
 	--n-tabs-direction: column;
 	--n-tabs-gap: 10px;
-	--n-tabs-border-width: 1px;
```
Comment (hein-obox) [inline modules/nested-tabs/assets/scss/frontend.scss:16]: These values had duplicates in this scss file.

File: modules/nested-tabs/assets/scss/frontend.scss [hunk 2]
```diff
@@ -105,6 +101,15 @@
 			height: var( --n-tabs-title-height );
 			width: var( --n-tabs-title-width );
 
+			&:not( .e-active ) {
+				background-color: var( --n-tabs-title-background-color );
+
+				&,
+				& a {
```
Comment (hein-obox) [inline modules/nested-tabs/assets/scss/frontend.scss:108]: I realised that the tab title text element gets an `a href`-element inside the front end, which can cause clashes with the default link styling. 

image

Comment (davseve) [reply modules/nested-tabs/assets/scss/frontend.scss]: The appears when the `Accessibility Improvements` experiment is inactive.
Good catch!!

File: tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js [hunk 3]
```diff
@@ -563,6 +563,100 @@ test.describe( 'Nested Tabs tests', () => {
...
+		// Normal tab styling: text color green, border color: green and icon color: yellow.
+		await editor.activatePanelTab( 'style' );
+		// Set text color.
+		await setTabItemColor( page, editor, 'section_title_style', 'title_normal', 'title_text_color', colorGreen );
+		// Set border color.
+		await setTabBorderColor( page, editor, 'normal', '', colorGreen, '5' );
+		// Set icon color.
+		await editor.activatePanelTab( 'content' );
+		await setTabItemColor( page, editor, 'icon_section_style', 'icon_section_normal', 'icon_color', colorYellow );
+		await editor.activatePanelTab( 'content' );
```
Comment (davseve) [inline tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js:603]: I think that this line is redundant.

File: tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js [hunk 4]
```diff
@@ -563,6 +563,100 @@ test.describe( 'Nested Tabs tests', () => {
...
+
+		// Hover tab styling: text color: red, border color: red and icon color: pink.
+		// Set text color.
+		await setTabItemColor( page, editor, 'section_title_style', 'title_hover', 'title_text_color_hover', colorRed );
+		// Set border color.
+		await setTabBorderColor( page, editor, 'hover', '_hover', colorRed, '5' );
+		// Set icon color.
+		await editor.activatePanelTab( 'content' );
+		await setTabItemColor( page, editor, 'icon_section_style', 'icon_section_hover', 'icon_color_hover', colorPink );
+		await editor.activatePanelTab( 'content' );
```
Comment (davseve) [inline tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js:615]: I think that this line is redundant.

**Review by davseve — CHANGES_REQUESTED** [review]

File: tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js (hunk 3 above)
Comment (hein-obox) [reply tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js:603]: @davseve I added these lines, to solve a glitch. Somehow the test struggled to switch between icon/title/tab panels. I added these lines to 'reset' to the original position.

**Review by davseve — DISMISSED** [review]

**Review by TzviRabinovitch — DISMISSED** [review]

**Review by TzviRabinovitch — APPROVED** [review]

**Pull request merged**